### PR TITLE
Release note improvements

### PR DESCRIPTION
**Pull request by carlpulley-da**
CHANGELOG_BEGIN
CHANGELOG_END

### CODE REVIEW

File: release/RELEASE.md [hunk 1]
```diff
@@ -147,32 +147,60 @@ how to communicate around it and when to remove the `prerelease` marker on the
...
 
-   On macOS/Linux:
+For example, for the Slack message:
+> team_daml_notifs
+>
+> Just published `2.4.0-snapshot.20220830.10494.0.4622de48`.
+>
+> For testing:
+>
+> ...
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:163]: If we're going to reference the Slack message, should we also note that it includes the (expanded) Windows link and the (expanded) curl command?

File: release/RELEASE.md [hunk 2]
```diff
@@ -182,145 +210,157 @@ This testing procedure starts once the release is listed on the [releases page].
...
+   > about 10 minutes (an easy way to check is to try to open `D:\` , as this volume is created after all the software 
+   > is installed).
    >
-   > All of the commands mentioned in this document can be run from a simple
+   > All of the commands mentioned in this testing section can be run from a simple
    > DOS prompt (start menu -> type "cmd" -> click "Command prompt").
+   > 
+   > At the end of your Windows testing session, please be sure to terminate the GCP instance by running 
+   > `./ad-hoc.sh destroy $ID`. Here `$ID` is the identity for your GCP instance - this is printed when you create your 
+   > Windows instance.
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:232]: Thanks for that :-)

File: release/RELEASE.md [hunk 3]
```diff
@@ -182,145 +210,157 @@ This testing procedure starts once the release is listed on the [releases page].
...
 
-1. Prerequisites for running the tests:
+3. Prerequisites for running the tests:
```
Comment (garyverhaegen-da) [inline release/RELEASE.md:234]: Markdown processors are expected to put the right numbers there, so keeping them all to `1.` makes it easier to refactor (adding, removing, swapping, etc.).

**Review by garyverhaegen-da — APPROVED** [review]
Thank you!

File: release/RELEASE.md (hunk 3 above)
Comment (carlpulley-da) [reply release/RELEASE.md:236]: It was an IntelliJ IDE thing! I'll undo its madness.

File: release/RELEASE.md (hunk 2 above)
Comment (S11001001) [reply release/RELEASE.md:234]: Perhaps we should recommend `temp` instead of `create`?

Comment (garyverhaegen-da) [reply release/RELEASE.md:234]: Good point! I keep forgetting that exists.

**Pull request merged**